content/business-models/none.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "None"
+date: 2019-01-09
+weight: 1
+---
+
+This is typically the ur-form of sustainable Free and Open Source Communities.
+The business model is to *not have a business model*. That is okay, because
+not every sustainable open source project needs or wants a business model.
+
+In this case the project has set up means and rules of collaboration, has a
+formal membership, usually without a formal position of authority or any other
+hierarchy but does **not** deal with monetary contributions.
+
+Sustainability in this non-model focuses on collaborator growth (quality and
+quantity) and shared responsibility to address it's biggest risk: contributor
+churn.
+
+### Who uses it?
+
+* [sfosc](https://sfosc.org/)
+* [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
+* [zsh](http://zsh.org/)
+
+### When should it be used?
+
+In the early phase of projects when it's not clear that the idea/implementation
+is a viable option in the free software eco system. Or when it's clear from the
+beginning, that the target audience is rather limited in size.
+
+Sometimes this model is also used as a way out of entrenched corporate
+influence. Incubating a new project, while blacking out monetary interests,
+trying to supersede the fought-over-project by technical means. Kind of a fork
+of the problem space (not the technology), resetting the relationship of the
+involved parties. A prominent example for this would be systemd or the GNOME
+desktop environment.
+
+### What kind of monetization is possible?
+
+Cash flow is avoided in this model because there is no legal entity behind the
+project (just individuals) that can handle the added complexity (control,
+reporting, taxes etc.) that monetization brings.
+
+The individuals involved cover the development costs for the project by
+donating their time. Infrastructure costs are avoided by utilizing free to use
```

Comment:
Would it be useful to say "donating their time (and any trivial expenses)"?

Reply:
I think that's covered in the line below: "or by donations of goods by members (like paying for a domain name etc.)"